src/bundle/DependencyInjection/ContainerBuilder.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\CompatibilityLayer\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder as SymfonyContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+
+/**
+ * @internal
+ */
+class ContainerBuilder extends SymfonyContainerBuilder
+{
+    public const EXTENSION_NAME_BC_MAP = [
```

Comment:
not needed atm, but think about extracting it to separate file as this array will grown a lot when we provide BC for another bundles.

Reply:
> not needed atm, but think about extracting it to separate file as this array will grown a lot when we provide BC for another bundles.

Yeah, probably need to change it, but would like to do it once the list is indeed too big. Also would like to avoid doing it in `__construct` as I'm not sure how much of BC there's gonna be for that constructor across Symfony versions.